### ISSUE TITLE
Localization support for audio (sound or music) resources

### ISSUE THREAD

**Issue by CelticMinstrel** [wesnoth/wesnoth#2987]
In 1.14.0, basic support was added in `[message]` for a voiceover of the message contents. This is an obvious candidate for localization. Unfortunately, the existing binary asset localization system is specific to image assets and is not trivially generalizable to audio. (It's also apparently broken in master, but that's a separate issue.)

What I would suggest is to generalize that system to work for any asset using a `[binary_path]` (though I think that also includes textdomains, but supporting it there should be harmless). Voiceovers in 1.14.0 are considered sound assets, so there definitely needs to be localization for sounds; but in principle it could also make sense to want localized music if a song has vocals. (I don't think it's very likely that anyone would go to the effort of doing this, mind you, but there's no reason to prevent it if it's no extra effort to allow it.)

**Comment by gfgtdf** [comment]
well at least as a workaroudn one could just make the key that spoecifies the file translatable?audio=_"myfile.wav" and a translator that implements the new tranlated audo file would then need to translate the "myfile.wav" aswell.

**Comment by CelticMinstrel** [comment]
Sure, I guess that's possible, but I think it's probably more work for the translators to do it that way...

**Comment by CelticMinstrel** [comment]
Huh, I forgot about this issue; #3935 addresses it.
